### PR TITLE
D8CORE-455 added more tools and styles to wysiwyg

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added more buttons to the html editor

# Needed By (Date)
- soon

# Urgency
- low

# Steps to Test
follow steps in https://github.com/SU-SWS/acsf-cardinalsites/pull/22

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by pookmish** [comment]
Tests should pass once https://github.com/SU-SWS/stanford_media/pull/21 is merged.

File: config/install/editor.editor.stanford_html.yml [hunk 1]
```diff
@@ -52,7 +56,7 @@ settings:
       linkit_enabled: true
       linkit_profile: default
     stylescombo:
-      styles: "a.su-button|Button\r\na.su-button--big|Big Button\r\na.su-button--secondary|Secondary Button"
+      styles: "a.su-button|Button\r\na.su-button--big|Big Button\r\na.su-button--secondary|Secondary Button\r\np.su-intro-text|Intro Text\r\np.su-font-splash|Splash Font\r\np.su-quote-text|Quote Text\r\np.su-drop-cap|Drop Cap First Letter\r\np.su-related-text|Related Text\r\np.callout-text|Callout Text"
```
[the remark this answers is not in the document]
Comment (pookmish) [reply config/install/editor.editor.stanford_html.yml]: Adding a test to check if those is visible is not really necessary. the kernel test checks to make sure the config is installed correctly. I attempted to create a behat test for it, but the dropdown options are exposed in an iframe with a dynamic `id` attribute. making it very difficult to switch into the iframe and check for text occurance.

[the remark this answers is not in the document]
Comment (sherakama) [reply config/install/editor.editor.stanford_html.yml]: Ok, checking the config should suffice.

**Comment by pookmish** [comment]
yay it passed

**Pull request merged**